packages/front-end/components/Features/RuleModal.tsx

```diff
@@ -75,6 +88,11 @@ export default function RuleModal({
       submit={form.handleSubmit(async (values) => {
         const ruleAction = i === rules.length ? "add" : "edit";
 
+        // If the user has the feature schedule toggle off, skip the formatting below.
+        if (!values.applyScheduleRules) {
```

Comment:
I think this is the correct approach. However, it would be good to not submit values.applyScheduleRules to the API if possible. Could just do something like
```
let payload = cloneDeep(values);
delete values.applyScheduleRules;
```

Reply:
Or if you want to keep the form values clean from this temporary variable, you could just hoist [applyScheduleRules, setApplyScheduleRules] to the RuleModal and pass in the wrapped setter function to ScheduleInputs props.

Reply:
In the latest update, I've done away with `applyScheduleRules` and instead am using the value of `scheduleToggleEnabled` in it's place here. So if the user defined 1 or both `scheduleRules`, but the toggle was disabled at the time of submission, we're ignoring the schedule rules.